### PR TITLE
removed a bunch of vulnerabilities

### PR DESCRIPTION
**Pull request by Decappi**

### CODE REVIEW

File: SS14.Client.Graphics/texture/TextureCache.cs [hunk 1]
```diff
@@ -5,8 +5,11 @@ namespace SS14.Client.Graphics.TexHelpers
 {
     public class TextureInfo
     {
-        public Texture Texture;
-        public Image Image;
+        private Texture texture;
+        private Image image;
+
+        public Texture Texture { get => texture; set => texture = value; }
+        public Image Image { get => image; set => image = value; }
```
Comment (YotaXP) [inline SS14.Client.Graphics/texture/TextureCache.cs]: Instead of writing properties and backing fields, why not just use auto-properties?
```diff
- public Texture Texture;
+ public Texture Texture {get; set;}
```

**Review by ZoldorfTheWizard — APPROVED** [review]

**Review by ZoldorfTheWizard — CHANGES_REQUESTED** [review]
Appveyor not happy

**Review by ZoldorfTheWizard — APPROVED** [review]

**Review by PJB3005 — CHANGES_REQUESTED** [review]
Auto-implemented properties, please.

File: SS14.Client.Graphics/Render/RenderImage.cs [hunk 2]
```diff
@@ -37,7 +37,7 @@ public class RenderImage : IRenderTarget, IDisposable
         /// SourceAlpha           == SrcAlpha
         /// InverseSourceAlpha    == OneMinusSrcAlpha
         /// </summary>
-        private BlendMode blendSettings = BlendMode.Alpha;
```
Comment (PJB3005) [inline SS14.Client.Graphics/Render/RenderImage.cs]: Now this doesn't default to `BlendMode.Alpha` anymore. You can just re-add the `= BlendMode.Alpha` before the semicolon.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**